template.yaml

```diff
@@ -485,6 +500,51 @@ Resources:
         - Key: Environment
           Value: !Ref DeploymentEnvironment
 
+  S3Bucket:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      BucketName: !Sub "jfrog-xray-aws-failed-findings-${DeploymentEnvironment}-${AWS::Region}"
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+      LifecycleConfiguration:
+        Rules:
+          - Id: GlacierRule
```

Comment:
Let's name this correctly. Not glacier at all. This is rule for standard_ia storage class. Same with the `Prefix`.

Reply:
Leftovers, removed, thanks!